TACC/tup-ui
# Task/tup 432 wording update remove user

## Overview:

To user removeremove buttons toggle and meet design specs

## Related:

- [TUP-432](https://jira.tacc.utexas.edu/browse/TUP-432)

## Changes:
Now toggles Remove Users with Confirm and Cancel options

## Testing:

1. Go to Project Listing page of a project you are either a PI or delegate of [http://localhost:8000/portal/projects](http://localhost:8000/portal/projects)
2. Click remove user
3. Make sure it has the same style as grant and publications with options Confirm Removal | Cancel
4. Make sure removing and cancelling still works

## UI:
When remove is selected for a user
![Screen Shot 2023-02-24 at 1 52 17 PM](https://user-images.githubusercontent.com/96220951/221277525-7325eacf-a1aa-4957-bbad-4bf359c127be.png)

When remove not selected for a user
![Screen Shot 2023-02-24 at 1 52 57 PM](https://user-images.githubusercontent.com/96220951/221277694-2adb11c6-d0c7-4ed7-a3b4-a3c4f77ae5ed.png)


## Notes:
